trello/api/features/negative/CardNegative.feature

```diff
@@ -0,0 +1,59 @@
+# Created by Oscar Lopez at 2/10/2020
+Feature: Negative test of Card
+  As a regular user, It manages the board, and user Gets a Error message Error.
+
+  Scenario: Lists actions on a card with an incorrect idCard
```

Comment:
If this scenario is for getting a card by invalid idCard, change the name of scenario to **Gets a card by incorrect idCard**